webrtc-stats.html

```diff
@@ -1996,6 +1996,23 @@ <h2>
                   level = -round(log10(audioLevel) * 20), with audioLevel 0.0 and values above 127
                   mapped to 127.
                 </p>
+                <p>
+                  The <a>audioLevel</a> represents the output audio
+                  level of the track; thus, if the track is sourced
+                  from an <a>RTCReceiver</a>, does no audio
```

Comment:
nit: RTCRtpReceiver

Reply:
did this metric not report on the input and an output audio level?

Reply:
Or at least there should be a section on input audio level as well.

Reply:
#271